README.md

```diff
@@ -518,6 +521,23 @@ Use `scripts/make_template.py` to dump 1D/2D histogram from `.coffea` to `TH1D/T
 </p>
 </details>
 
+## Submit standalone condor jobs
+
+You have the option to run the framework through "standalone condor jobs", bypassing the native coffea-supported job submission system. Within each job, a standalone script will execute the following on the worker node:
+
+ - Set up a minimal required Python environment.
+ - Retrieve the BTVNanoCommissioning repository, either from a git link or transferred locally.
+ - Launch the `python runner.py ...` command to execute the coffea framework in the iterative executor mode.
```

Comment:
this step is only meant as a local test of your workflow before submitting condor jobs? Because in the next step, you mention using the wrapper script `condor/submitter.py` instead of `runner.py` itself?

Reply:
Thanks! let's leave as separate script